### PR TITLE
Adding Fider - No Database

### PR DESCRIPTION
**Pull request by aqidd**
First of all, thank you for your contribution! 😄

### ☑️ Self Check before Merge

- [x] I have tested the template using the method described in README.md thoroughly
- [x] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [x] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [x] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [x] Icon is added as a png file to the logos directory.

### CODE REVIEW

File: public/v4/apps/fider.yml [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+            EMAIL_SMTP_PORT: $$cap_smtp_port
+            EMAIL_SMTP_USERNAME: $$cap_smtp_user
+            EMAIL_SMTP_PASSWORD: $$cap_smtp_pass
+        caproverExtra:
+            containerHttpPort: '3000'
+caproverOneClickApp:
+    variables:
+        - id: $$cap_db_url
+          label: Fider database URL
+          description: Database connection URL for Fider.
```
Comment (githubsaturn) [inline public/v4/apps/fider.yml:22]: This is really unclear. Maybe add an example:
```
Postgres database connection URL for Fider: example: "postgres://username:password@srv-captain--mypostgres:5432/fider?sslmode=disable"
```

File: public/v4/apps/fider.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        documentation: Taken from https://hub.docker.com/r/getfider/fider/
```
Comment (githubsaturn) [inline public/v4/apps/fider.yml:4]: There is not much documentation on dockerhub. Maybe link this one? https://getfider.com/docs/hosting-instance

**Review by githubsaturn — COMMENTED** [review]
Looks good! Thanks!

Just a few minor comments.

**Comment by aqidd** [comment]
Thanks for the review @githubsaturn , all incorporated 🚀

**Comment by githubsaturn** [comment]
Thanks!

**Comment by ak4zh** [comment]
@aqidd Can you please update the fider app tp support the latest image.

`HOST_DOMAIN` is changed to `BASE_URL`

In latest `getfider/fider:stable`  there is an issue all assets are being looked at `https://{BASE_URL}/{BASE_URL}/actual-path`

**Comment by aqidd** [comment]
https://github.com/caprover/one-click-apps/pull/822

**Pull request merged**